arches/app/media/js/views/components/workflows/new-tile-step.js

```diff
@@ -35,6 +35,7 @@ define([
         this.alert = params.alert || ko.observable(null);
         this.complete = params.complete || ko.observable();
         this.completeOnSave = params.completeOnSave === false ? false : true;
+        this.altButtons =  params.altButtons || ko.observable(null);
```

Comment:
I think that `altButtons` should perhaps instead be two properties: one to hide the default buttons and one for additional buttons.  This would allow for reuse of the existing buttons alongside additional ones as well as make the templates slightly simpler